code/game/atoms.dm

```diff
@@ -783,7 +781,6 @@
 				if(fingerprintslast != H.ckey)
 					fingerprintshidden += "\[[all_timestamps()]\] (Wearing gloves). Real name: [H.real_name], Key: [H.key]"
 					fingerprintslast = H.ckey
-				H.gloves.add_fingerprint(M)
```

Comment:
Is this removal a mistake? I wasn't added with your blood PR and has been around for years.

Reply:
This caused touching something to call add_fibers twice. 
This is what caused gloves spreading blood infinitely in current master.
This was calling add_blood from gloves to themself if they are covered in blood.
This is just weird behaviour that shouldnt be here, touching something shouldnt add fibers from you to your gloves. You anyway leave fibers on them when you pick them up, put them on.